tests/components/airly/test_config_flow.py

```diff
@@ -0,0 +1,92 @@
+"""Define tests for the Airly config flow."""
+import json
+
+from airly.exceptions import AirlyError
+from asynctest import patch
+from tests.common import load_fixture, MockConfigEntry
```

Comment:
Please sort the tests.* import group below homeassistant import group.

Reply:
Done